### PR TITLE
Beta garmon (because feature isolation is for chumps)

### PR DESCRIPTION
**Pull request by The4D6**
# Tasks:

### New Units

- [x] Endryd Haar

### Oaths
- [x] An Eternity of War
- [x] In Disgrace All Are equal
- [x] Only In Death Does Duty End
- [x] Panoply of Old
- [x] Pride is our Armour
- [x] The Broken Helix
- [x] The Eternal Vendetta
- [x] The Flesh Is Week
- [x] The Legacy of Nikaea
- [x] The Spoils of Victory
- [x] The Taint of Xenos
- [x] The Weapons of Desperation

# This PR does not validate properly in BattleScribe due to the lack of ability to check upstream values

Also on this branch:
Solar Aux New Units: #3135
 Horus Aximand - Sons of Horus
 Tybalt Marr - Sons of Horus
 Legion Optae - LA, Everybody can take!
 Legion Company Command Squad - LA, Everybody can take!

Recon Company

### CODE REVIEW

**Comment by nstephenh** [comment]
This PR also has all the changes from my  units wave 1 PR so that should also be tested.
First suggestion, we should move blackshield oaths up in the xml so it's next allegiance/rite of war. 
We also talked about hiding rite of war and I don't think we did it, the change would be to uncheck the export flag (I think it's called import) in the blackshields cat and then manually import it in all the space marine cats.

File: 2022 - Horus Heresy.gst
Comment (nstephenh) [inline 2022 - Horus Heresy.gst:1]: Two copies of blackshield primary and optional detachment

**Comment by The4D6** [comment]
> This PR also has all the changes from my units wave 1 PR so that should also be tested. First suggestion, we should move blackshield oaths up in the xml so it's next allegiance/rite of war. We also talked about hiding rite of war and I don't think we did it, the change would be to uncheck the export flag (I think it's called import) in the blackshields cat and then manually import it in all the space marine cats.

I don't want to go making too many more changes, and I think messing about with Rites of War is outside the scope of this PR, they're all hidden from Blackshields anyway. Happy to make the change in a future piece of work however

**Comment by Mayegelt** [comment]
The WoD update seems to be a roster miamer for Battlescribe users. It throws up errors on units using the "collective" option.
This isn't just on Blackshield armies, but impacts all legion armies. The only way to resolve seems to be deleting units or deleting collective models for those units and adding in a bunch of individuals like we have for Termys. This is a major knockback for QoL for Battlescribe users.
I can't really see a great alternative outside of remaking all the units for a Blackshields WoD variant of each of the units. It wouldn't be hard to copy paste each of them. But would of course mean that any updates that aren't legion specific (not including legion specific wargear that of course can be taken by both in theory) would need to also remember to be copied into the copy as well.

image

image

image

Test roster used for above picys as proof
https://cdn.discordapp.com/attachments/638905973481930773/1241446222384988293/IW_test.rosz?ex=664a3a64&is=6648e8e4&hm=450567f96eba6456fc269875ff7c795e4d34f4add16286e39e85e18fadb7bd9d&

**Comment by nstephenh** [comment]
Legacy of Nikea, psycher sub-type is not selected by default, and the warp blast is 'hidden' but selected and that's throwing an error:
image

**Comment by The4D6** [comment]
> Legacy of Nikea, psycher sub-type is not selected by default, and the warp blast is 'hidden' but selected and that's throwing an error: image

It shouldn't be selected by default, it's optional per-character

**Comment by nstephenh** [comment]
Merging, we're just "Sending it"

**Pull request merged**